static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -81,6 +81,7 @@ define([
         var win         = (opts || {}).window || window,
             viewId      = (opts || {}).viewId,
             page        = config.page,
+            ophanHelper = config.ophan,
```

Comment:
why helper? maybe consider being consistent and calling it ophan everywhere (or ophanHelper everywhere, but I'm not a fan of ambiguous words like helper)